cypress/e2e/edit-page.cy.js

```diff
@@ -10,8 +10,8 @@ describe('Edit a form page', function () {
       .should('have.value', 'What is your work address?')
   })
 
-  it('allows the user to navigate back to the form overview page', function () {
-    cy.contains('Go to form overview').click()
-    cy.url().should('eq', `${Cypress.config().baseUrl}/forms/${this.formId}`)
+  it('allows the user to navigate back to the forms  page', function () {
```

Comment:
Might help to name this the same as above for consistency:
```suggestion
  it('allows the user to navigate back to the form pages', function () {
```